presto-docs/src/main/sphinx/functions/ipaddress.rst

```diff
@@ -0,0 +1,9 @@
+===================
+Ipaddress Functions
+===================
+
+.. function:: ip_prefix(ip_address, prefix_bits) -> ipprefix
+
+    Returns the IP prefix of a given ``ip_address`` with subnet size of ``prefix_bits``. ``ip_address`` can be either of type ``VARCHAR`` or type ``IPADDRESS``. ::
```

Comment:
Is the line a bit too long?

Reply:
![image](https://user-images.githubusercontent.com/33299678/66782297-3bf01c80-ee8a-11e9-93e5-6c2a97d00edf.png)

The line wrapping happens by default so it's fine.

Reply:
@yingsu00  - the general try is to wrap in the `rst` file around 80 chars and then the UI takes care of correctly showing it. So this is more of a style guide.

Reply:
It's a comment about coding style not generated documentation format. Just make sure the width is reasonable.

Reply:
Should we also specify that the return type is of type `IPPREFIX`

Reply:
> Should we also specify that the return type is of type `IPPREFIX`

This is indicated in the `function` part: ` -> ipprefix`.

Reply:
> @yingsu00 - the general try is to wrap in the `rst` file around 80 chars and then the UI takes care of correctly showing it. So this is more of a style guide.

I see thanks for the explanation!  How about this? The first line is at 89 chars but I saw plenty of examples the lines are a bit over 80 chars.
```
    Returns the IP prefix of a given ``ip_address`` with subnet size of ``prefix_bits``.
    ``ip_address`` can be either of type ``VARCHAR`` or type ``IPADDRESS``. ::

        SELECT ip_prefix(CAST('192.168.255.255' AS IPADDRESS), 9); -- {192.128.0.0/9}
        SELECT ip_prefix('2001:0db8:85a3:0001:0001:8a2e:0370:7334', 48); -- {2001:db8:85a3::/48}```